src/coreclr/src/jit/gentree.h

```diff
@@ -5991,23 +5991,35 @@ struct GenTreePhiArg : public GenTreeLclVarCommon
 
 struct GenTreePutArgStk : public GenTreeUnOp
 {
+private:
+    unsigned m_byteOffset;
+#ifdef FEATURE_PUT_STRUCT_ARG_STK
+    unsigned m_byteSize; // The number of bytes that this argument is occupying on the stack with padding.
+#endif
+
+public:
+#if defined(DEBUG_ARG_SLOTS)
```

Comment:
IIUC this will be going away, so that we won't retain the redundancy of `gtSlotNum` with `m_byteOffset`. Is that right?

Reply:
Yes,  I keep the old fields just for asserts during the transition.